lib/plugins/platform/platform.test.js

```diff
@@ -37,20 +37,23 @@ describe('Platform', () => {
   describe('#archiveService()', () => {
     let getAuthTokenStub;
     let archiveServiceRequestStub;
+    let logStub;
 
     beforeEach(() => {
       archiveServiceRequestStub = sinon
         .stub(platform, 'archiveServiceRequest')
         .resolves({ data: {} });
-      sinon.spy(console, 'log');
+      getAuthTokenStub = sinon.stub(platform, 'getAuthToken');
+      logStub = sinon.stub(platform.serverless.cli, 'log');
     });
 
     afterEach(() => {
-      console.log.restore();
+      platform.archiveServiceRequest.restore();
```

Comment:
@pmuens is the restore necessary in any way? Each test will run in isolation and then stub it anyway?

Reply:
Yes, that's true. But what if some dev removes a stub and another test later on relies on the side-effect that the stub is restored?

Reply:
**Edit:** You're right 💯

Sorry I haven't read the code properly you've commented on. I thought that you were referring to another `stub` which was refactored in this commit.